examples/robolectricExample/build.gradle

```diff
@@ -0,0 +1,51 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'de.undercouch:gradle-download-task:2.0.0'
+    }
+}
+
+apply plugin: 'de.undercouch.download'
+apply plugin: 'android-sdk-manager'
+apply plugin: 'com.android.application'
+apply plugin: 'android-command'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'realm-android'
+apply plugin: 'realm-robolectric'
```

Comment:
I think most developers does not need the support for Robolectric for Darwin. They can use other unit test tools, they can use Mockup objects or even they don't use any unit test tool. So I want this as a sepate plugin not to bother other developers.